socialhome/streams/app/components/StreamElement.vue

```diff
@@ -6,126 +6,66 @@
             v-bind="author"
             :is-user-author="isUserAuthor"
             :is-user-local="isUserLocal"
-            :current-browsing-profile-id="currentBrowsingProfileId"
-            :is-user-authenticated="isUserAuthenticated"
         >
         </author-bar>
-        <div class="grid-item-bar d-flex justify-content-start">
+        <reactions-bar
+            :id="id"
+            :replies-count="repliesCount"
+            :shares-count="sharesCount"
+            :has-shared="hasShared"
+        >
             <div class="mt-1">
-                <a :ref="contentUrl" :title="timestamp">
+                <a :href="contentUrl" :title="timestamp" variant="link" class="unstyled-link">
                     {{ humanizedTimestamp }}<span v-if="edited"> {{ editedText }}</span>
                 </a>
-                <div v-if="isUserAuthor">
-                    <a :href="updateUrl">
+                &nbsp;
+                <template v-if="isUserAuthor">
```

Comment:
What does the `template` tag do here, why that instead of `div`?

Reply:
The `template` tag will not be rendered. Only what it contains. It's usefull to do [condiftionnal groups](https://vuejs.org/v2/guide/conditional.html#Conditional-Groups-with-v-if-on-lt-template-gt).